README.md

```diff
@@ -3,6 +3,9 @@
 This repo contains samples that demonstrate
 the API used in Windows classic desktop applications.
 
+> [!NOTE
```

Comment:
Typo here? (Missing close bracket?)

Reply:
Ah, that might be why it didn't work. I'll fix it. I'll do it directly though, not via PR, if that's ok.

Reply:
No, that only works for docs.microsoft.com markup. Reverting to simper formatting.